source/reference/failure-recovery.txt

```diff
@@ -0,0 +1,96 @@
+.. _c2c-failure-recovery:
+
+================
+Failure Recovery 
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+:program:`mongosync` temporarily alters some collection characteristics
+during synchronization. The original values are restored during the
+:ref:`commit process <c2c-api-commit>`. However, a network failure or
+some other problem could disrupt synchronization before the sync is
+committed. If the sync ends prematurely, the temporary changes must be
+manually updated on the destination cluster.
+
+Manually revert the following temporary changes:
+
+- :ref:`Unique indexes set as non-unique <c2c-dr-unique>`.
+- :ref:`TTL indexes with expireAfterSeconds set to MAX_INT
+  <c2c-dr-ttl>`.
+- :ref:`Hidden indexes set as non-hidden <c2c-dr-hidden>`.
+- :ref:`Capped collections with size/max set to the maximum possible
+  value <c2c-dr-capped>`.
+
+.. _c2c-dr-unique:
+
+Unique Indexes
+--------------
+
+:program:`mongosync` replicates writes in parallel which can cause
+transient uniqueness violations on the destination cluster. To avoid
+these errors, unique indexes are replicated as non-unique. If
+synchronization ends early, the user must manually resolve any
+uniqueness violations and convert the non-unique indexes back to
+unique indexes.
+
+For each non-unique index on the destination cluster that should be
+unique, run the :dbcommand:`collMod` with ``prepareUnique`` set to
```

Comment:
[suggestion] I don't think a definite article is required here, because I think of database commands as proper nouns, but you could make the argument either way. 

"... run `collMod` with ``prepareUnique`` set to ``true``."

Reply:
Done